omnigibson/objects/stateful_object.py

```diff
@@ -32,16 +34,7 @@
 # Create module logger
 log = create_module_logger(module_name=__name__)
 
-
-# Optionally import bddl for object taxonomy.
```

Comment:
we remove this because we force bddl to be an explicit OG dependency now, right?

Reply:
sg